src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -45,30 +46,57 @@ function isValidFileType(fileName) {
   return FILE_TYPES.some(type => fileName.toLowerCase().endsWith(type));
 }
 
-function isValidImageSize(result) {
+// data urls start with data:image/jpg; and we need to pull out the
+// image/jpg part so canvas doesn't change the file type
+function getMimeType(dataUrl) {
```

Comment:
Could we use file.type to get the media type? And in the validation helper as well? It seems like the downside of only being able to rely on this for standard types is not a problem here.

Reply:
Ha, I looked for something like that and totally missed it. Yes, that would be better.

Reply:
I'm not sure about using it for the validation helper. I think that would make the code more complicated, since we'd need to map the type to the file suffix (e.g. image/jpeg is for both .jpeg and .jpg).

Reply:
It looks like there may be some additional extensions that we may wish to account for, [bmp](https://tools.ietf.org/html/rfc7903)(dib), [jpeg](https://www.sitepoint.com/mime-types-complete-list/)(jpe).